components/org.wso2.carbon.identity.media.core/src/main/java/org/wso2/carbon/identity/media/core/file/FileBasedStorageSystemImpl.java

```diff
@@ -140,15 +135,80 @@ public boolean evaluateSecurity(String accessLevel, String id, String type, Stri
     }
 
     @Override
-    public void deleteFile(String id, String type, String tenantDomain) throws StorageSystemException {
+    public boolean evaluateDownloadSecurityForProtectedMedia(String id, String type, String tenantDomain) throws
```

Comment:
Shall we change the name to start with `is` since this returning a boolean?

Reply:
fixed all similar places with: https://github.com/wso2/identity-media/pull/13/commits/ce54589bc6783386823b033dcd3725ef1ac3615c